app/src/main/java/com/gh4a/fragment/RepositoryFragment.java

```diff
@@ -148,8 +140,49 @@ public void onActivityCreated(Bundle savedInstanceState) {
         fillData();
         setContentShown(true);
 
-        getLoaderManager().initLoader(0, null, mReadmeCallback);
-        getLoaderManager().initLoader(1, null, mPullRequestsCallback);
+        mLoadPullRequestsCount = result -> {
+            Log.d("TEST", "getPullRequest onNext called: " + result);
+            View v = getView();
+            v.findViewById(R.id.issues_progress).setVisibility(View.GONE);
+            v.findViewById(R.id.pull_requests_progress).setVisibility(View.GONE);
+
+            TextView tvIssuesCount = (TextView) mContentView.findViewById(R.id.tv_issues_count);
+            tvIssuesCount.setText(String.valueOf(mRepository.getOpenIssues() - (Integer) result));
+
+            TextView tvPullRequestsCountView = (TextView) v.findViewById(R.id.tv_pull_requests_count);
+            tvPullRequestsCountView.setText(String.valueOf(result));
+        };
+
+        mReadmeConsumer = result -> { // Fill-in readme
+            Log.d("TEST-RE", "readme flatMap called");
+            setContentShown(true);
+            TextView readmeView = (TextView) mContentView.findViewById(R.id.readme);
+            View progress = mContentView.findViewById(R.id.pb_readme);
+            if (result != null)
+                mImageGetter.encode(getContext(), mRepository.getId(),
+                        String.valueOf(result));
+
+            if (result != null) {
+                readmeView.setMovementMethod(UiUtils.CHECKING_LINK_METHOD);
+                mImageGetter.bind(readmeView, String.valueOf(result), mRepository.getId());
+            } else {
+                readmeView.setText(R.string.repo_no_readme);
+                readmeView.setTypeface(Typeface.DEFAULT, Typeface.ITALIC);
+            }
+            readmeView.setVisibility(View.VISIBLE);
+            progress.setVisibility(View.GONE);
+        };
+
+        // Fetch Readme file + PullRequests Count
+        loadReadme()
+                .compose(RxTools.handle(getActivity(), RepositoryService.LOAD_README))
+                .subscribe(mReadmeConsumer, e -> {
```

Comment:
Perhaps move these subscribe calls inside of `loadReadme`, `loadPullRequestCount` methods? Then there will be no need to create consumer fields.

Reply:
That way I'd have to modify the functions `signature` in order to add a `boolean` parameter indicating wether to **refresh** the data, which would result in such call as `loadReadme(true)` or `loadReadme(false)` according to the need for **fresh / cached** data

Reply:
I would prefer that myself.

Reply:
I like that too actually, gonna take care of it ;)